src/tests/ftest/SConscript

```diff
@@ -0,0 +1,41 @@
+"""Build ftest"""
+import os
+
+def scons():
+    """Execute build"""
+    Import('env', 'prereqs')
+
```

Comment:
dirs = ['container', 'control', 'daos_test', 'data', 'io',
            'network', 'object', #you get the idea ]

for dir in dirs:
      env.Install(os.path.join(ftest_install_dir, dir),
                        Glob("%s/*.*" % dir))
 Or perhaps Phil can improve it even more.  Regardless, no reason to duplicate